src/plugins/gauge/components/Gauge.vue

```diff
@@ -343,14 +390,28 @@ export default {
         dialLowLimitDeg() {
             return this.percentToDegrees(this.valToPercent(this.limitLow));
         },
+        meterOorRangeIndicatorAspectRatio() {
```

Comment:
I think this has a typo

Reply:
Assume you mean the "Oor"? If so, not a typo: that's for "out of range". Looking at it now, I think I should do that bit in all caps, rather than camel.

Reply:
Got it!

Reply:
Though shouldn’t it be meterOORangeIndicatorAspectRatio then? I’d add a comment specifying what the OO stands for so dummies like me won’t bother you in the future about it 😄

Reply:
I've renamed this to `meterOutOfRangeIndicatorAspectRatio`